apps/03-efectos-colaterales/src/effects.erl

```diff
@@ -8,13 +8,31 @@
 %% @spec print(N :: pos_integer()) -> ok
 %% @end
 %%--------------------------------------------------------------------
-print(_N) ->
-    erlang:throw(not_implemented).
+print(0) ->
```

Comment:
`pos_integer()` does not include 0.
https://www.erlang.org/docs/25/reference_manual/typespec